core/build.rs

```diff
@@ -113,6 +137,7 @@ fn main() {
     let mut builder = cc::Build::new();
     builder.opt_level(3);
     builder.pic(true);
+    builder.flag("-mssse3");
```

Comment:
Is this added due to the DPDK version change? (Shouldn't native arch be fine?)

Reply:
No, I just needed this to be able to build retina on NixOS. I guess `-march=native` doesn't work on NixOS for reproducability reasons. Its harmless on other OSes.

Reply:
It will break on ARM :)

Reply:
I removed the line. I can add it in my local tree when bulding for NixOS. Btw, it would be nice to have an ARM CI.

Reply:
Agree! there's a bug open on this, but we haven't yet been able to prioritize it.